addons/sale_timesheet/static/src/scss/project_project.scss

```diff
@@ -0,0 +1,5 @@
+@include media-breakpoint-down(md) {
+    .o_project_form_view .o_notebook_content .o_kanban_view .o_kanban_ghost {
+        display: none;
+    }
+}
```

Comment:
I am wondering if we should not do that in generic way since it seems we have "padding bottom" in all kanban view displayed in x2many field. 🤔 

If yes, then we could maybe do something like: 

```scss
diff --git a/addons/web/static/src/views/kanban/kanban_controller.scss b/addons/web/static/src/views/kanban/kanban_controller.scss
index 4b94184363ab..4de669851574 100644
--- a/addons/web/static/src/views/kanban/kanban_controller.scss
+++ b/addons/web/static/src/views/kanban/kanban_controller.scss
@@ -2,6 +2,10 @@
 .o_kanban_view {
     @include media-breakpoint-down(md) {
         --ControlPanel-border-bottom: none;
+
+        &.o_field_x2many_kanban .o_kanban_ghost {
+            display: none;
+        }
     }
 }
```

What do you think? 🙂

Reply:
If we do that, then can you add it in a separate commit. 🙂 

Thanks in advance! 😁